packages/backend/src/lib/errors.ts

```diff
@@ -94,6 +94,42 @@ export const SPECIFIC_ERRORS = {
         "failed-precondition",
         "Unable to interact with a multi-part upload (start, create pre-signed urls or complete).",
         "Authenticated user is not a current contributor which is currently in the uploading step."
+    ),
+    SE_CONTRIBUTE_NO_CEREMONY_CIRCUITS: makeError(
+        "not-found",
+        "There is no circuit associated with the ceremony.",
+        "No documents in the circuits subcollection were found for the selected ceremony."
+    ),
+    SE_CONTRIBUTE_NO_OPENED_CEREMONIES: makeError("not-found", "There are no ceremonies open to contributions."),
```

Comment:
I think this is missing one of the 3 descriptions

Reply:
Yes, the third one (details) is optional. In this particular error, is not needed at all.